test/integration/subscription_test.exs

```diff
@@ -6,9 +6,25 @@ defmodule Braintree.Integration.SubscriptionTest do
   alias Braintree.Customer
   alias Braintree.Subscription
 
+  def create_test_subscription do
+    {:ok, customer} = Customer.create(%{payment_method_nonce: "fake-valid-nonce"})
+    [card] = customer.credit_cards
+    Subscription.create(%{payment_method_token: card.token, plan_id: "starter"})
+  end
+
   test "create/1 with a plan_id" do
     assert {:ok, customer} = Customer.create(%{payment_method_nonce: "fake-valid-nonce"})
     [card] = customer.credit_cards
-    assert {:ok, _subscription } = Subscription.create(%{payment_method_token: card.token, plan_id: "starter"})
+    assert {:ok, _subscription} = Subscription.create(%{payment_method_token: card.token, plan_id: "starter"})
+  end
+
+  test "find/1 with a subscription_id" do
+    {:ok, subscription} = create_test_subscription
+    assert {:ok, _subscription} = Subscription.find(subscription.id)
```

Comment:
Something should be asserted about the subscription that comes back, beyond the fact that it is `ok`. For example, that the result is a `%Subscription{}` and that the `plan_id` is "starter".